WEB/Src/DependencyCollector/DependencyCollector/Implementation/AzureSdk/AzureSdkDiagnosticsEventHandler.cs

```diff
@@ -38,6 +38,13 @@ public override void OnEvent(KeyValuePair<string, object> evnt, DiagnosticListen
         {
             try
             {
+                if (SdkInternalOperationsMonitor.IsEntered())
+                {
+                    // Now that Application Insights supports AAD, we need to check if an internal operation is being caught here.
+                    // type = "InProc | Microsoft.AAD"
+                    return;
+                }
+
```

Comment:
@cijothomas , this appears to fix the InternalOperation issue. Please review. :)

Reply:
yes. this is right approach. previously there was no need of this, as SDK's own internal operation never triggered it.
(if SDK makes a SQLTelemetryChannel, then we need to modify Sqllistener to have this check :))